content/docs/for-app-developers/concepts/experimental-features.md

```diff
@@ -0,0 +1,27 @@
++++
+title="What are experimental features?"
+weight=8
++++
+
+Certain features are considered `experimental` and susceptible to change in a future API version.
+
+<!--more-->
+
+This means users will need to enable the `experimental` mode in order to use one of these features.
+
+To enable these features, run `pack config experimental true`, or add `experimental = true` to the `~/.pack/config.toml` file.
+
+For example, exporting your application to disk in `OCI` layout format is an experimental feature available on `pack` since version `v0.30.0`.
+
+The following is a list of experimental features:
+
+* Image extensions
+* Windows extensions
+* Windows containers
+* Windows buildpackage
+* Buildpack registry
+* Flattening a buildpack package
+
+For more information and to look at an example of how this might be valuable, see [Export to OCI layout format on disk][exp-feature].
```

Comment:
```suggestion
The following features are experimental for `pack`:
* Building for [Windows containers][windows]
* Packaging buildpacks for Windows containers
* Exporting to [OCI layout][oci-layout] format on disk
* Interacting with the [buildpack registry][registry]
* Creating a multi-arch buildpack package (`pack manifest` commands)
* Flattening a buildpack package (`pack buildpack --flatten`)
* Interactive builds (`pack build --interactive`)
* When building, reading project metadata version & source URL from [project.toml][project-descriptor]

The following features are experimental for `lifecycle`:
* Building for [Windows containers][windows]
* Exporting to [OCI layout][oci-layout] format on disk
```

Reply:
@AidanDelaney and my code suggestions crossed paths, but this might be a bit more comprehensive & accurate (extensions no longer experimental)

Reply:
updated